pageobjects/homepage.js

```diff
@@ -0,0 +1,92 @@
+var page = require('./page');
+var assert = require('assert');
+//var prop = require('./prop');
+
+//Page objects
+var saleButton = '//*[@class="b7afbb84"]'
+var locationBar = '//*[@placeholder="Location"]'
```

Comment:
I think this locator will not work once you select any location filter. Can you verify this?